src/main/java/org/protege/editor/owl/client/diff/ui/ReviewButtonsPanel.java

```diff
@@ -244,7 +251,20 @@ public void actionPerformed(ActionEvent e) {
         	 
         }
     };
-    
+
+    private ActionListener restartPelletBtnListener = new ActionListener() {
+        @Override
+        public void actionPerformed(ActionEvent actionEvent) {
+            ClientSession sess = ClientSession.getInstance(editorKit);
+            LocalHttpClient client = (LocalHttpClient) sess.getActiveClient();
+
+            if (!client.isWorkFlowManager(sess.getActiveProject())) return;
+
+            // TODO: implement this
+//            pelletService.restart();
```

Comment:
Looks good. Is there a problem with injecting pelletService and just calling it here like this commented code?